### DISCUSSION TITLE
Combobox's inputValue not changing with rerender

### DISCUSSION THREAD

**Discussion by chjpark** [twilio-labs/paste#772] [Q&A] [answered]
Hi, 

I'm using the Combobox component and I would like it to be able to change the inputValue when I change the state of the inputValue. I have my props change the inputValue but the comboBox doesn't actually show the change. It still shows the value as the previous value. Is there anything else I need to do in order to set the inputValue when I change the state of the value I want to pass in?

**Comment by SiTaggart** [comment, marked as answer]
Hi there,

There is a section on the combobox docs about Combobox being used as a controlled components https://paste.twilio.design/components/combobox#controlled-combobox. Followed by a section on how to fully control the state of the component using the state hook.

In the mean time a reproduction of your issue in codesandbox would be very beneficial to understanding the issue you are facing./

Thanks
